services/QuillLMS/app/workers/retrieve_google_classrooms_worker.rb

```diff
@@ -1,12 +0,0 @@
-# frozen_string_literal: true
-
-class RetrieveGoogleClassroomsWorker
```

Comment:
Note, this deploy will likely caused some errored jobs (since some of these jobs will be queued), but we can manually re-run these.

Reply:
I will monitor and run those errored ones manually.